src/notebook/notebookKernel.ts

```diff
@@ -112,6 +113,40 @@ export class JuliaKernel {
         }
     }
 
+    private async containsJuliaEnv(folder: string) {
+        return (await exists(path.join(folder, 'Project.toml')) && await exists(path.join(folder, 'Manifest.toml'))) ||
+            (await exists(path.join(folder, 'JuliaProject.toml')) && await exists(path.join(folder, 'JuliaManifest.toml')))
```

Comment:
Probably a pointless microoptimization, but `readdir` seems like the better choice.

Reply:
I'm not even sure? I guess kind of depends how `exists` is implemented, right? But we'd have to handle casing manually, folders with a very large number of files etc. Maybe also all easy, but for now this works :)